src/pages/payment-methods/_mobile-expanded-list.js

```diff
@@ -86,10 +84,6 @@ const StyledRefLink = styled(Flex)`
     @media ${device.mobileL} {
         justify-content: flex-start;
     }
-
-    @media ${device.tabletM} {
-        width: 65%;
```

Comment:
Removing this part as we needed it before because the sentence was longer and would be broken in tablet mode.